rts/System/Input/MouseInput.cpp

```diff
@@ -68,6 +69,9 @@ IMouseInput::~IMouseInput()
 
 bool IMouseInput::HandleSDLMouseEvent(const SDL_Event& event)
 {
+	if (!mouse->ButtonPressed() && RmlGui::ProcessMouseEvent(event)) {
```

Comment:
why the `!mouse->ButtonPressed()` check?

Reply:
Only want to give a mouse event to RmlUi if the mouse isn't currently performing a drag. Otherwise box selections get stuck when the mouse goes over an Rml element.

Reply:
I've taken the liberty of putting that as a comment in the code ca9aeb0baa3fff5727eb87fc6c5588f856586144